src/NServiceBus.Core/EndpointConfiguration.cs

```diff
@@ -114,17 +113,8 @@ public void UseContainer(IContainer builder)
             ContainerComponent.UseContainer(builder);
         }
 
-        /// <summary>
-        /// Specifies the range of types that NServiceBus scans for handlers etc.
-        /// </summary>
-        internal void TypesToScanInternal(IEnumerable<Type> typesToScan)
```

Comment:
there are actually quite a few locations where we might use reflection to get access to this. (https://github.com/search?q=org%3AParticular+TypesToScanInternal&type=Code) Did you have a look at those scenarios?

Reply:
Good point, all usages seems to be in tests and to generate docs but I will move it back and add a test to make sure we don't move it

Reply:
Done https://github.com/Particular/NServiceBus/pull/5494/commits/008113a2b3ea832d05c024acaae0b94c23912246